tests/contrib/django/django1_app/settings.py

```diff
@@ -3,6 +3,7 @@
 
 ALLOWED_HOSTS = [
     "testserver",
+    "localhost",
```

Comment:
Do we need to allow localhost? Looking at the new test (test_get_asgi_span) the client should be running on testserver

Reply:
Added it here just to support running the Django app locally outside of the container